src/mrinufft/trajectories/maths.py

```diff
@@ -0,0 +1,313 @@
+"""Utility functions for mathematical operations."""
+
+import numpy as np
+import numpy.linalg as nl
+
+
+CIRCLE_PACKING_DENSITY = np.pi / (2 * np.sqrt(3))
+
+
+##########
+# PRIMES #
+##########
+
+
+def compute_greatest_common_divider(p, q):
+    """Compute the greatest common divider of two integers p and q.
+
+    Parameters
+    ----------
+    p : int
+        First integer.
+    q : int
+        Second integer.
+
+    Returns
+    -------
+    int
+        The greatest common divider of p and q.
+    """
+    while q != 0:
+        p, q = q, p % q
+    return p
```

Comment:
I guess this could be replaced with [`np.gcd`](https://numpy.org/doc/stable/reference/generated/numpy.gcd.html) ?

Reply:
Indeed, the function is removed and its single use has been replaced & checked